ci/test.sh

```diff
@@ -58,7 +58,11 @@ run_tests() {
             sleep $[60 + (${RANDOM} % 30)]
         fi
 
-        ${TIMEOUT_CMD} ${TIMEOUT_TIME_ARG} ${TIMEOUT_SEC-${TIMEOUT_DEFAULT}} ./core_test
+        if [ -d /c/ ]; then
+            ${TIMEOUT_CMD} ${TIMEOUT_TIME_ARG} ${TIMEOUT_SEC-${TIMEOUT_DEFAULT}} ./core_test.exe
+        else
+            ${TIMEOUT_CMD} ${TIMEOUT_TIME_ARG} ${TIMEOUT_SEC-${TIMEOUT_DEFAULT}} ./core_test
```

Comment:
Instead of repeating the whole command, use an executable suffix variable (see above)